main.go

```diff
@@ -53,6 +54,13 @@ func main() {
 			serviceConfig.Name = name
 			serviceConfig.Source = source
 
+			serviceConfig.Environment = v.GetString(f.Service.Deployer.Eventer.GitHub.Environment)
+			serviceConfig.HTTPClientTimeout = v.GetDuration(f.Service.Deployer.Eventer.GitHub.HTTPClientTimeout)
+			serviceConfig.OauthToken = v.GetString(f.Service.Deployer.Eventer.GitHub.OauthToken)
+			serviceConfig.Organisation = v.GetString(f.Service.Deployer.Eventer.GitHub.Organisation)
+			serviceConfig.PollInterval = v.GetDuration(f.Service.Deployer.Eventer.GitHub.PollInterval)
+			serviceConfig.ProjectList = v.GetStringSlice(f.Service.Deployer.Eventer.GitHub.ProjectList)
```

Comment:
We don't pass around viper and flags fully, I passed around the actual variables. Passing around viper probably makes life easier.